src/components/change-view.jsx

```diff
@@ -134,15 +137,52 @@ export default class ChangeView extends React.Component {
 
     renderSubmission () {
         if (!this.props.user) {
-            return <div>Log in to submit annotations.</div>;
+          return <div>Log in to submit annotations.</div>;
         }
 
-        const markAsSignificant = () => console.error('markAsSignificant not implemented');
-        const addToDictionary = () => console.error('addToDictionary not implemented');
+        const annotation = this.state.annotation || {};
+
+        let markSignificant;
+        if (annotation.significance >= 0.5) {
+          markSignificant = <span className="status lnk-action">Significant</span>;
+        }
+        else {
+          markSignificant = (
+            <span className="lnk-action">
+              <i className="fa fa-upload" aria-hidden="true" />
+              <button
+                className="btn btn-link"
+                disabled={this.state.addingToImportant}
```

Comment:
Now that we aren’t using Typescript, we don’t need to initialize anything there. But I probably should have.